msbuild/custom-task-code-generation/README.md

```diff
@@ -131,29 +165,57 @@ Then, the dependencies of your MSBuild task must be packaged inside the package,
         <TargetFramework>netstandard2.0</TargetFramework>
         <version>1.0.0</version>
         <title>AppSettingStronglyTyped</title>
-        <authors>John Doe</authors>
+        <authors>John</authors>
         <description>Generates a strongly typed setting class base on a txt file</description>
         <tags>MyTags</tags>
-        <copyright>Copyright ©Microsoft Company 2022</copyright>
+        <copyright>Copyright ©Contoso 2022</copyright>
         <!-- we need the assemblies bundled, so set this so we don't expose any dependencies to the outside world -->
-        <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+        <GenerateDependencyFile>true</GenerateDependencyFile>
         <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
         <DebugType>embedded</DebugType>
         <IsPackable>true</IsPackable>
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.0.0" />
+      <PackageReference
+        Include="Microsoft.Build.Utilities.Core"
+        Version="17.0.0"
+        PrivateAssets="all" />
+      <PackageReference
+        Include="Microsoft.Extensions.DependencyInjection"
+        Version="6.0.0"
+        PrivateAssets="all" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <!-- These lines pack the build props/targets files to the `build` folder in the generated package.
+         By convention, the .NET SDK will look for build\<Package Id>.props and build\<Package Id>.targets
+         for automatic inclusion in the build. -->
+        <Content Include="build\AppSettingStronglyTyped.props" PackagePath="build\" />
+        <Content Include="build\AppSettingStronglyTyped.targets" PackagePath="build\" />
     </ItemGroup>
 
     <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="ResolveReferences">
         <ItemGroup>
-            <!-- the dependencies of your MSBuild task must be packaged inside the package, they cannot be expressed as normal PackageReferences -->
+            <!-- The dependencies of your MSBuild task must be packaged inside the package, they cannot be expressed as normal PackageReferences -->
 
             <!--example: <BuildOutputInPackage Include="$(PkgFParsec)/lib/netstandard2.0/FParsecCS.dll" />-->
```

Comment:
in this README, as well as in the csproj for the app-settings task, the `CopyProjectReferencesToPackage` target is missing the line that actually copies the PackageReference's runtime assets into the package. That line is:

```xml
<BuildOutputInPackage
      Include="@(ReferenceCopyLocalPaths)"
      TargetPath="%(ReferenceCopyLocalPaths.DestinationSubPath)" />
```